bbr.Dockerfile

```diff
@@ -15,10 +15,10 @@ COPY go.mod go.sum ./
 RUN go mod download
 
 # Sources
-COPY cmd ./cmd
+COPY cmd/bbr ./cmd
 COPY pkg ./pkg
```

Comment:
Copying the complete package here, because bbr needs the following packages
handlers/server.go:30:	logutil "sigs.k8s.io/gateway-api-inference-extension/pkg/epp/util/logging"
handlers/server.go:31:	requtil "sigs.k8s.io/gateway-api-inference-extension/pkg/epp/util/request"

I can copy these individually, but the same might not look clean. Hence, copying in the complete package, 
@danehans , please let me know, if I have update this copy only individual directories. 
Thank you!